vultr/resource_vultr_instance.go

```diff
@@ -126,11 +124,10 @@ func resourceVultrInstance() *schema.Resource {
 				Optional: true,
 			},
 			"reserved_ip": {
```

Comment:
Should `reserved_ip` should be `reserved_ip_id` ?

Reply:
good catch - I'll push that change up